src/styles/init/reset.css

```
@@ -5,10 +5,28 @@
   margin: 0;
   padding: 0;
 
+  font: inherit;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)